test/suites/integration/bsp/missing-chunks.test.ts

```diff
@@ -3,7 +3,7 @@ import { describeBspNet, registerToxic, type EnrichedBspApi } from "../../../uti
 // TODO: Add asserts to this test case when we impl the missing chunks handling
 describeBspNet(
   "BSP: Missing Chunks",
-  { initialised: false, networkConfig: "noisy" },
+  { initialised: false, networkConfig: "noisy", skip: true },
```

Comment:
Why skip?

Reply:
I originally did this out of laziness since the test case is placeholder.

But now i've refactored toxiproxy so that it works as expected with both bspnet and fullnet. it's become an injected element if you provide a flag `NOISY=1` as opposed to its own compose file.